### ISSUE TITLE
WAI IG charter https://www.w3.org/WAI/IG/20200804.html

### ISSUE THREAD

**Issue by brewerj** [w3c/strategy#228]
Draft updated charter for discussion in Strategy on August 4, and for horizontal review

**Comment by samuelweiler** [comment]
[Link to draft charter](https://www.w3.org/WAI/IG/20200804.html)
[diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FWAI%2FIG%2Fcharter6&doc2=https%3A%2F%2Fwww.w3.org%2FWAI%2FIG%2F20200804.htm)

The charter is missing the full history table, and the few words about history are copied from the previous charter, which means they're no longer correct: "Differences between this charter and the previous charter for this Group include increasing the emphasis on accessibility issues throughout W3C work."

**Comment by samuelweiler** [comment]
Headers refer to "initial team contacts" and "initial chair", when it appears those have changed over time: https://www.w3.org/WAI/IG/charter4.html

**Comment by samuelweiler** [comment]
I wish the old charters had pointers forward to the next-newest charters, e.g. from https://www.w3.org/WAI/IG/charter4.html to https://www.w3.org/WAI/IG/charter5.html.  example: https://www.w3.org/2011/07/privacy-ig-charter

**Comment by samuelweiler** [comment]
Where should minor editorial suggestions go? e.g. is the charter in Github?  If not, I suggest https://github.com/w3c/charter-drafts

**Comment by samuelweiler** [comment]
I see no privacy or security issues here.  I have some additional minor editorial suggestions, and I'll wait for the answer to [my question above](https://github.com/w3c/strategy/issues/228#issuecomment-675513708) for where to send those.

It is fascinating to see how W3C charters have changed over time, particularly noting how very brief this one is.  I'm particularly struck by the simplicity of membership - unlike PING, which people can join in the usual ways, it looks like the WAI IG has none of the DB backing - it's just a mailing list.

Other than the history table, which I think is valuable, I wonder if there is anything substantive in the [current charter template](https://w3c.github.io/charter-drafts/charter-template.html) that needs to be included in the charter.

As an outsider, it would help me if this sentence in the deliverables section were expanded to name the groups involved.  They're named in the next section, but not with the context of who does what.
> ... nor replace the review and development which specific accessibility-related W3C groups are chartered to do.

**Comment by brewerj** [comment]
Sam, thanks for the privacy and security review. Replies follow on other issues: 

- Updating history: ack, aware this needs updating, that is one of the pending clean-ups as mentioned in Strat today.
- "Initial team contact" and "Initial chair" are generally relative to a proposed/renewed charter, not to an entire charter sequence. It is there in case there is a mid-charter change of TC or chairs, as these (especially chairs) are typically updated in place in the case where there is a change during a charter's term. But I will check in case the "Initial xyz" is no longer part of the expected template.
- Minor editorial suggestions are welcome here.
- Not DB-backed, correct; but allowable, and it serves a useful cross-group purpose for a large community.
- Charter trends have indeed changed over time; this one went through a cycle of encouraged tersification by some AC members, to make charters easier to review. 
- Ack on the charter template as well; that is another pending check based on an earlier comment. 
- I'll clarify which groups do what.

**Comment by michael-n-cooper** [comment]
APA thinks "throughout W3C" at the top of the mission implies some overlap with APA scope, which includes outreach to W3C groups. Context later in the charter addresses this, but this could lead to confusion. APA review complete, over to @brewerj to complete accessibility horizontal review.

**Comment by r12a** [comment]
It seems to me that it would be useful for the WAI IG to track the work taking place in the i18n WG, since WAI has some recommendations related to i18n, and has other recommendations that may be affected by i18n considerations. Therefore a sentence in the charter to say that the IG will monitor i18n work would seem to be relevant  in the dependencies & liaisons section.

**Comment by himorin** [comment]
No further comment/request from i18n WG.

**Comment by brewerj** [comment]
Richard, thanks for this review comment. The WAI Interest Group doesn't have a structured activity to monitor work of other W3C WGs, whereas the Accessible Platform Architectures (APA) WG does. Also APA WG already has an established dependency with the I18n WG; and has primary responsibility for reviewing non-WAI W3C groups' technical work on behalf of all W3C accessibility groups. You can see that here: [APA WG Coordination](https://www.w3.org/2018/08/apa-charter#w3c-coordination) Please let me know if that addresses your concern. 

The mission in the WAI IG proposed charter does refer to "this includes augmenting wide review and discussion of accessibility aspects of deliverables being developed in W3C groups." In practice, this amplification of wide review of accessibility aspects of W3C deliverables is focused on amplifying review of accessibility groups' deliverables, as listed in the coordination section 3.1. 

I18n is welcome to post deliverables on the WAI IG list for feedback at any point, but we encourage you to rely on formal coordination with the APA WG. Please let me know whether this clarification satisfies I18N's concerns.

**Comment by brewerj** [comment]
Michael, thanks for the APA review of the proposed WAI IG renewed charter. You mentioned: 
> APA thinks "throughout W3C" at the top of the mission implies some overlap with APA scope, which includes outreach to W3C groups.

The intention is for WAI IG to serve as a public face of some of the more technical WAI groups, and for the WAI IG to help draw interested participants into specific areas of WAI technical work. This is repeated later as a bullet in the Scope section: 
"promoting awareness of and engagement in accessibility-related work throughout W3C groups;"
Does that part seem sufficiently clear? 

Here's the originally proposed language: 
> This includes augmenting wide review and discussion of accessibility aspects of deliverables being developed in W3C groups, including specifications, research topics, and educational materials; exploring web accessibility issues and solutions; and sharing information....
Here's a minor clarification: 
> This includes augmenting wide reviews of WAI deliverables, and augmenting APA WG's review and discussion of accessibility aspects of deliverables being developed in W3C groups, including specifications, research topics, and educational materials; exploring web accessibility issues and solutions; and sharing information;....
Does that address APA's concern? 
Otherwise please suggest alternative language.

**Comment by r12a** [comment]
hi @brewerj.  When i made my comment, I wasn't really thinking about a formal review activity.  Just that you might want to list i18n as a group with overlapping interests. But i's only a suggestion.

**Comment by brewerj** [comment]
@r12a, thanks for the clarification, I see your point. I try something different after updating the template, as the language in the coordination section may need to change again then. Will ping you then.

**Comment by brewerj** [comment]
Confirming accessibility horizontal review completed, clarification about APA WG role accepted.

**Comment by brewerj** [comment]
@JAlanBird requested revisiting staff effort; reviewed; left SLH at 5% but reduced JB effort to 3%.

**Comment by brewerj** [comment]
Hi @r12a , fyi, I added this sentence to the "Coordination" section; does this address your suggestion? 

"Internationalization Working Group, to explore intersections between accessibility and internationalization"

[Charter](https://www.w3.org/WAI/IG/20200804.html)

**Comment by wseltzer** [comment]
Rechartered: https://lists.w3.org/Archives/Public/public-new-work/2021Jan/0001.html
